components/card/stories/card.stories.js

```diff
@@ -1,11 +1,10 @@
-import { html } from "lit";
 
 // Import the component markup template
 import { Template } from "./template";
 
 import { default as ActionButton } from "@spectrum-css/actionbutton/stories/actionbutton.stories.js";
-import { default as QuickAction } from "@spectrum-css/quickaction/stories/quickaction.stories.js";
 import { default as Checkbox } from "@spectrum-css/checkbox/stories/checkbox.stories.js";
+import { default as QuickAction } from "@spectrum-css/quickaction/stories/quickaction.stories.js";
```

Comment:
Do you have thoughts on how we would deprecate this option within Card since QuickAction will go away?

Reply:
Yeah, once design documents the deprecation, we'll map that deprecation noticed in our docs site and remove the option from storybook.

Reply:
Oh very cool! How does design deprecate components, does that happen in their guidelines? Is there an example of that I could peek at for context?

Reply:
This is how they handle it: https://spectrum.adobe.com/page/quick-actions/

I tried to mimic it as much as made sense.